index.ts

```diff
@@ -1,19 +1,19 @@
 import { Inngest } from "inngest";
 import { serve } from "inngest/lambda";
 
-const inngest = new Inngest({ name: "Inngest Lambda Example" });
+const inngest = new Inngest({ id: "inngest-lambda-example" });
 
 const helloWorld = inngest.createFunction(
-  { name: "Hello World" },
+  { name: "Hello World", id: "hello-world" },
   { event: "test/hello.world" },
   async ({ event, step }) => {
-    await step.sleep("1s");
+    await step.sleep("wait-for-a-sec", "1s");
     return "Hello World";
-  }
+  },
 );
 
-const optionalConfigForLocalTesting = {
-    ...(process?.env?.INNGEST_REGISTER_URL && {inngestRegisterUrl: process.env.INNGEST_REGISTER_URL}),
-}
```

Comment:
This can be removed as now v3 has a standard `INNGEST_BASE_URL` supported by the SDK for local development with the dev server.